src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -31,6 +31,7 @@ const SECTIONS = new Map([
   ['network', 'Network'],
   ['notifications', 'Notifications'],
   ['help-and-feedback', 'Help and feedback'],
+  ['theme', 'Appearance'],
```

Comment:
I think we can remove this from this PR, since we won't have this tab in the preferences, but it is a good idea 😄